syntax = "proto3";

message State {
  bool switch = 1;
  bool ephemera = 2;
  int32 brightness = 3;
  bool daytime = 4;
}

message Params {
  int64 clock_interval = 1;
}
